vagrant-ansible/roles/ruby/tasks/main.yml

```diff
@@ -15,19 +15,19 @@
   when: user_exists|failed
 
 - name: clone rbenv from github
-  sudo_user: "{{ rbenv_user }}"
+  become_user: "{{ rbenv_user }}"
   git: "repo=git://github.com/sstephenson/rbenv.git dest=~/.rbenv accept_hostkey=yes"
 
 - name: add rbenv path to .profile
-  sudo_user: "{{ rbenv_user }}"
+  become_user: "{{ rbenv_user }}"
   lineinfile: dest=~/.profile line='export PATH="$HOME/.rbenv/bin:$PATH"'
 
 - name: add eval rbenv init to .profile
-  sudo_user: "{{ rbenv_user }}"
+  become_user: "{{ rbenv_user }}"
   lineinfile: dest=~/.profile line='eval "$(rbenv init -)"'
 
 - name: clone ruby-build from github
-  sudo_user: "{{ rbenv_user }}"
+  become_user: "{{ rbenv_user }}"
   git: "repo=git://github.com/sstephenson/ruby-build.git dest=~/.rbenv/plugins/ruby-build accept_hostkey=yes"
 
 - name: check if specific version of ruby is already installed
```

Comment:
今回の修正とは関係がないかもしれませんが、このタスクでエラーになります。

```
TASK [ruby : check if specific version of ruby is already installed] ***********

fatal: [bitkids_dev]: FAILED! => {"changed": false, "cmd": "sudo -iu vagrant rbe
nv versions | grep -q 2.3.0", "delta": "0:00:00.103929", "end": "2016-03-17 13:5
0:24.851568", "failed": true, "rc": 1, "start": "2016-03-17 13:50:24.747639", "s
tderr": "", "stdout": "", "stdout_lines": [], "warnings": ["Consider using 'beco
me', 'become_method', and 'become_user' rather than running sudo"]}
```